### ISSUE TITLE
Handlebars interpolation not working

### ISSUE THREAD

**Issue by jptillman** [mbest/knockout.punches#70]
Versions in use: punches 0.5.1, knockout 3.4.1

I've added knockout.punches to an existing ko project and some features seem to function, such as filters in a 'text' binding handler, but the handlebars interpolation is not.  I just see the literal value after the bindings are applied, i.e., `{{objValue}}` remains just that in the output.

I've checked that `interpolationMarkupPreprocessor()` is really getting added via `addNodePreprocessor()` but `provider.preprocessNode` never gets called!

How do I go about troubleshooting further?

**Comment by jptillman** [comment]
Hm.  Never mind.  Years ago, I had followed the little trick documented [here](http://www.knockmeout.net/2013/06/knockout-debugging-strategies-plugin.html) under the heading **Catching exceptions using a custom binding provider** which replaced the bindingProvider property with a custom one.  So I just had to pull in the original bindingProvider's preprocessNode property to get it to work.

Sorry for the false alarm.
